app/views/core/Root.vue

```diff
@@ -2,6 +2,18 @@
     <router-view></router-view>
 </template>
 
+
 <script>
-  export default {}
+  export default {
+    metaInfo: function () {
+      return {
+        title: 'CodeCombat - Learn how to code by playing a game',
+        titleTemplate: '%s | CodeCombat',
```

Comment:
Unrelated but brining up for discussion given this is touching titles.  Do we want to change the postfix in CN?

Reply:
Good question for @nwinter

Reply:
Followed up with Nick - this is fine for now.